packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/sagas.ts

```diff
@@ -119,7 +119,11 @@ export default ({ api, coreSagas, networks }) => {
     yield call(fetchUser)
   }
 
-  const defineSteps = function* (tier, needMoreInfo) {
+  const contextMapper = function* (origin = 'DEFAULT_CONTEXT') {
+    return yield origin === 'BuySell' ? 'FIAT_DEPOSIT' : 'DEFAULT_CONTEXT'
```

Comment:
It seems that this could be manage with a Enum object. I see several times this constant hardcoded